website/static/dist/devicescript-vm.js

```diff
@@ -988,28 +988,28 @@ function ignoredModuleProp(prop) {
   }
 }
 
-function unexportedMessage(sym, isFSSybol) {
+function unexportedMessage(sym, isFSSymbol) {
   var msg = "'" + sym + "' was not exported. add it to EXPORTED_RUNTIME_METHODS (see the FAQ)";
-  if (isFSSybol) {
+  if (isFSSymbol) {
```

Comment:
This correction doesn't appear outside this file which argues it's being imported from somewhere else.. Happy to drop or queue sending a PR to it...